Add definition for "business days" in the code review page (#442)
mattermost/mattermost-developer-documentation: site/content/contribute/getting-started/code-review.md

    * When in doubt, ask for help in the [Developers](https://community.mattermost.com/core/channels/developers) channel on our community server.
3. Wait for a review.
    * The ticket author, a core committer, or a product manager will assign reviewers and label your pull request appropriately.

>     * Expect some interaction with at least one reviewer within 5 "business days" (weekdays - Monday through Friday - other than major holidays concurrent in US and Canada).

Reviewer: Suggested tweak, but I'll also defer to @justinegeffen for thoughts:
```suggestion
    * Expect some interaction with at least one reviewer within 5 business days (weekdays, Monday through Friday, excluding [statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays))
```

Reviewer 2: Thanks, @lieut-data! Two minor changes ("5" > "five") and a period at the end of the sentence. Otherwise that reads clearly to me! :)

Reviewer 3: As someone not living on the american continent, I'm wondering if it does make more sense to rephrase this into something like

```suggestion
    * Expect some interaction with at least one reviewer within 5 "business days" (weekdays - Monday through Friday). Be aware that there might be local holidays.
```

Author: > Thanks, @lieut-data! Two minor changes ("5" > "five") and a period at the end of the sentence. Otherwise that reads clearly to me! :)

@lieut-data, @justinegeffen thanks for the feedback!

* Shouldn't we update the referenced definition in [Feature Release Process](https://docs.mattermost.com/process/feature-release.html) to be consistent with the suggested change above?

* While I personally think digits like "5" are easier to visually scan than letters "five", I think it is best to be consistent regardless. So, should we update all the digits in the page to letter form?

Author: > As someone not living on the american continent, I'm wondering if it does make more sense to rephrase this into something like

Thanks @hanzei for the suggestion! As a fellow non-Americas habitat 😄, I think it is best to be explicit which holidays are we referring to, and that was what discussed during the devs meeting. Makes sense?

Reviewer 2: @abdusabri, I actually agree with you that it's easier to scan but stylistically it should be written out. Taking that into account I am happy to review the larger doc and make similar changes. 

WRT the feature release process docs, they should definitely match this wording, which means we should change "business days" to "working days". 

@hanzei, I see your point. The use of "local holidays" may be a bit confusing as customers may not know which local holiday is being observed. Mentioning US and Canada gives a bit more context and is easier to Google.

Reviewer 2: So my suggestion is: 

"* Expect some interaction with at least one reviewer within 5 "working days" (weekdays - Monday through Friday - other than major holidays concurrent in US and Canada).

Author: > So my suggestion is:
> 
> "* Expect some interaction with at least one reviewer within 5 "working days" (weekdays - Monday through Friday - other than major holidays concurrent in US and Canada).

Thanks @justinegeffen for your feedback!

I will update the PR according to your suggestion (using working days) once @lieut-data & @hanzei confirm.

Author: > > So my suggestion is:
> > "* Expect some interaction with at least one reviewer within 5 "working days" (weekdays - Monday through Friday - other than major holidays concurrent in US and Canada).
> 
> Thanks @justinegeffen for your feedback!
> 
> I will update the PR according to your suggestion (using working days) once @lieut-data & @hanzei confirm.

@hanzei, @lieut-data your feedback/confirmation would be highly appreciated 🙂

Reviewer 3: To me this document is a rough outline what the process looks like. I don't expect any one count the number of days until a PR is released. Hence, I also don't expect any one to google, if today is a US holiday to know, if his/her PR has been reviewed.

I would rather outline, that there might be local holidays:

```suggestion
    * Expect some interaction with at least one reviewer within 5 "business days" (weekdays - Monday through Friday). Please note that there might be local holidays.
```

Reviewer 4: I don't have the full context, and don't mean to jump in but I got the stale PR notification :) 

Wondering if we can come to an agreement on the wording. Tweaking @lieut-data's original proposal above:
```
Expect some interaction with at least one reviewer within 5 business days (weekdays, Monday through Friday, excluding [statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays)).
```

seems to capture the feedback above. Let me know if you have concerns @hanzei @justinegeffen @lieut-data @abdusabri 

Specifically:
 - Keep "5" to be consistent with the rest of this page, and other processes such as the feature release process. Propose a separate issue is opened if we want to write out single-digit words, as this would be a change across our docs and beyond the scope here
 - Use "business days", consistent with the rest of the page. Changes are beyond the scope here.
 - Jesse's reference to statutory holidays doesn't call out US and Canada specifically, and links to how statutory holidays are considered at Mattermost if someone wants to learn more. Changes on how local holidays are treated should be made to the handbook instead (https://docs.mattermost.com/process/working-at-mattermost.html#holidays), and beyond the scope here

Author: Thanks @jasonblais, appreciated!

I've no concerns.

Reviewer 3: :+1: for Jason's proposal

Reviewer 2: +1 for Jason's proposal. 

I agree that "5" to "five" is a separate topic altogether and will create a new issue for it. Happy to keep it as-is for now.

Reviewer 3: @abdusabri I think we have a critical mass to go forward with the proposal :wink: 

Would you please apply the change?
